clang/test/CAS/driver-cache-launcher.c

```diff
@@ -24,6 +24,29 @@
 // SESSION: "-cc1depscan" "-fdepscan=daemon" "-fdepscan-share-identifier"
 // SESSION: "-fcas-path" "[[PREFIX]]/cas" "-fcas-token-cache" "-greproducible"
 
+// RUN: cp -R %S/Inputs/cmake-build %t/cmake-build
+// RUN: pushd %t/cmake-build
+// RUN: clang-cache-build-session -prefix-map-cmake -v echo 2>&1 | FileCheck %s -check-prefix=SESSION-CMAKE-PREFIX
+// RUN: env CLANG_CACHE_CAS_PATH=%t/cas clang-cache-build-session -prefix-map-cmake clang-cache %clang -c %s -o %t.o -### 2>&1 | FileCheck %s -check-prefix=CLANG-CMAKE-PREFIX -DPREFIX=%t
+// RUN: popd
+
+// SESSION-CMAKE-PREFIX: note: setting CLANG_CACHE_PREFIX_MAPS=/llvm/build=/^build;/llvm/llvm-project/llvm=/^src;/llvm/llvm-project/clang=/^src-clang;/llvm/llvm-project/clang-tools-extra=/^src-clang-tools-extra;/llvm/llvm-project/third-party/benchmark=/^src-benchmark;/llvm/llvm-project/other/benchmark=/^src-benchmark-1;/llvm/llvm-project/another/benchmark=/^src-benchmark-2
```

Comment:
`CHECK-SAME` is not checking an exact continuation (it would miss it if an unexpected path showed up in-between).
I tried `CHECK-SAME-NEXT` or `CHECK-NEXT-SAME` and they both worked as `CHECK-SAME`.
Unless I'm missing something, checking the whole line was the way to ensure checking the exact set of paths that should show up.

(though I just realized that I forgot to add `{{$}}` at the end of the line, oops)